### ISSUE TITLE
Scroller not working in tablet

### ISSUE THREAD

**Issue by muzammil17** [bhuh12/vue-router-tab#99]
hey,
In my application router header is scrolling on desktop pc or laptop using mouse. But my application is for sales persons which will use application on tablets. I'm testing on **10 inch width tablet** and it is not working via using finger.
Please make it possible to scroll using fingers in tablets.
![router-tab](https://user-images.githubusercontent.com/30322831/101977297-8b88ce80-3c6e-11eb-929b-9c4f9fce276d.jpg)

**Comment by bhuh12** [comment]
Can you provide the browser's `navigator.userAgent`?
Whether swiping left or right will trigger the browser's back and forward actions?

**Comment by muzammil17** [comment]
My chrome version is up to date in both my pc and tablet. But it is still not working in tablet.
chrome version = **Version 87.0.4280.88 (Official Build) (64-bit)** for pc
tablet version = **Version 87.0.4280.101 (Official Build) (64-bit)**
navigator.userAgent
**Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/87.0.4280.88 Safari/537.36**

**Comment by bhuh12** [comment]
This doesn’t look like a tablet useragent.
https://bhuh12.github.io/vue-router-tab/demo/
Can the problem be reproduced on the Demo？

**Comment by muzammil17** [comment]
I did not changed any configuration of the package except for the css. I also checked by installing it again but tabs are not scrolling on tablet using touch. It is working on a mobile devices but not on tablet. Im using basic configuration of a package

this is what im getting by detecting my device. Screenshot attached below shows the user agent of a tablet.
![Screenshot (30)](https://user-images.githubusercontent.com/30322831/102177469-73f75300-3ec5-11eb-8c70-1f52d7998a28.png)

**Comment by bhuh12** [comment]
Is this an Android tablet?
The user agent did not provide enough information to distinguish this is a tablet device.

```
mounted() {
  if(isTablet) {
    const scroller = document.querySelector('.router-tab__scroll-container')
    scroller.classList.add('is-mobile')
  }
}
```

**Comment by bhuh12** [comment]
Can these codes solve the problem?

**Comment by muzammil17** [comment]
Im Really sorry for the late reply. 
I have placed the above codes in my app.vue file but it is giving errors. Can you please tell where do I have to writes these codes if not in app.vue
![Screenshot (38)](https://user-images.githubusercontent.com/30322831/103521372-75031b00-4e9a-11eb-9bb3-43004dd8b6dd.png)
![Screenshot (37)](https://user-images.githubusercontent.com/30322831/103521381-792f3880-4e9a-11eb-8054-33c3d3bedaee.png)

**Comment by muzammil17** [comment]
I have tested this application on two tablets
1. dtab d-01h huawei
2. Sony Xperia Tablet Z LTE

I also created a small project for you on sandbox
[test app](https://codesandbox.io/s/silly-blackburn-ynn4c)

And please also let me know about the above codes.

Thanks.

**Comment by bhuh12** [comment]
https://codesandbox.io/s/eloquent-forest-ptjx2?file=/src/components/home.vue

https://ptjx2.csb.app/child/2dd0f1c7-1f79-4655-b406-7c888b4cf262

```
<template>
  <div class="hello">
    <router-tab lang="en" />
  </div>
</template>

<script>
function getScrollbarWidth() {
  const div = document.createElement("div");
  const styles = {
    width: "100px",
    height: "100px",
    overflowY: "scroll",
  };
  for (let i in styles) div.style[i] = styles[i];
  document.body.appendChild(div);
  const scrollbarWidth = div.offsetWidth - div.clientWidth;
  div.remove();
  return scrollbarWidth;
}

export default {
  mounted() {
    const isMobile = !getScrollbarWidth();
    if (isMobile) {
      const scroller = document.querySelector(".router-tab__scroll-container");
      scroller && scroller.classList.add("is-mobile");
    }
  },
};
</script>

<style scoped></style>
```

**Comment by muzammil17** [comment]
Scroller is now working.
Thank You

**Comment by bhuh12** [comment]
The code to fix the problem has been released to `v1.2.3`. Please update and verify.

Changelog:
https://bhuh12.github.io/vue-router-tab/guide/changelog.html

**Comment by muzammil17** [comment]
Yes, it is working.
